src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerR5IT.java

```diff
@@ -65,25 +68,37 @@ public void testCreateAndRead() {
   @Test
   public void testWebsocketSubscription() throws Exception {
 
+	  String endpoint = "ws://localhost:" + port + "/websocket";
     /*
-     * Create topic (will be contained in subscription)
+     * Create topic
      */
     SubscriptionTopic topic = new SubscriptionTopic();
-    topic.setId("#1");
-    topic.getResourceTriggerFirstRep().getQueryCriteria().setCurrent("Observation?status=final");
 
-    /*
+	 topic.setUrl(SUBSCRIPTION_TOPIC_TEST_URL);
+	  topic.setStatus(Enumerations.PublicationStatus.ACTIVE);
+	  SubscriptionTopic.SubscriptionTopicResourceTriggerComponent trigger = topic.addResourceTrigger();
+	  trigger.setResource("Observation");
+	  trigger.addSupportedInteraction(SubscriptionTopic.InteractionTrigger.CREATE);
+	  trigger.addSupportedInteraction(SubscriptionTopic.InteractionTrigger.UPDATE);
+
+	 ourClient.create().resource(topic).execute();
+
+	  waitForSize(1, () -> ourClient.search().forResource(SubscriptionTopic.class).where(Subscription.STATUS.exactly().code("active")).cacheControl(new CacheControlDirective().setNoCache(true)).returnBundle(Bundle.class).execute().getEntry().size());
```

Comment:
Nitpick:  This is long and a bit hard to read without scrolling.  I suggest breaking this down to one line per method call